app/src/main/java/org/oppia/app/utility/Logger.kt

```diff
@@ -0,0 +1,197 @@
+package org.oppia.app.utility
+
+import android.util.Log
+import org.oppia.app.application.OppiaApplication
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+import java.io.IOException
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+
+/** Wrapper class for Android Log. */
+object Logger {
+
+  private val ENABLE_CONSOLE_LOG = true
+  private val ENABLE_FILE_LOG = true
+  private val GLOBAL_LOG_LEVEL = LogLevel.VERBOSE
+  private val LOG_DIRECTORY =
+    OppiaApplication.applicationContext().getExternalFilesDir(null).toString() + File.separator + "OppiaAppLog" + File.separator
+
+  private enum class LogLevel private constructor(val logLevel: Int) {
+    VERBOSE(Log.VERBOSE),
+    DEBUG(Log.DEBUG),
+    INFO(Log.INFO),
+    WARNING(Log.WARN),
+    ERROR(Log.ERROR),
+    ASSERT(Log.ASSERT)
+  }
+
+  /**
+   *
```

Comment:
Let's include what the method actually does, and we can probably avoid the extra ``@param`` documentation for simplicity:

``/** Logs a verbose message with the specified tag. */``

Ditto elsewhere.

Reply:
okay done